### ISSUE TITLE
Overlap matrix S(B) of perturbation-dependent atomic orbitals 

### ISSUE THREAD

**Issue by cyberturist** [pyscf/properties#6]
Dear devs,

thank you for this awesome module!
Is it possible to print the overlap matrix of gauge-invariant atomic orbitals S(B) in the calculation of NMR or magnetizabilities?

**Comment by sunqm** [comment]
No, the field-included integrals are not available. If you just need to compute the first order term, `mol.intor('int1e_igovlp')` can compute the summation of two overlap integrals $<e^{i R_p\times r \cdot B} \chi_p|\chi_q> + <\chi_p|e^{i R_q\times r \cdot B} \chi_q>$ .
